src/poc/miner_poc_grpc_client_statem.erl

```diff
@@ -0,0 +1,1080 @@
+-module(miner_poc_grpc_client_statem).
+-behavior(gen_statem).
+%% tmp disable no match warning from dialyzer
```

Comment:
Fix this?

Reply:
We have been trying buts its not playing ball.